c4_sign/base_task.py

```diff
@@ -299,3 +301,28 @@ def draw(self, canvas: Canvas, delta_time: timedelta):
                 return True
             return False
         return super().draw(canvas, delta_time)
+
+class JavaTask(ScreenTask):
```

Comment:
so, right now, simulator shows this as just `JavaTask`, and doesn't actually pull the name (which, for the python ones, is derived from from the class's name). might be worth adding in a thing or something for that?

Reply:
I've changed it to work base on the ScreenTask's title, now, at least for overriding the current task in the simulator and for sending the task's names to the simulator. Is there anywhere else that we'd need to change that? Should we be making a different field for this purpose than just the title of the ScreenTask?